task/06-conditions-n-loops-tasks.js

```diff
@@ -63,7 +77,18 @@ function getFactorial(n) {
  *   -1,1  =>  0  ( = -1 + 0 + 1 )
  */
 function getSumBetweenNumbers(n1, n2) {
-    throw new Error('Not implemented');
+    const length = n2 - n1 + 1;
+
+    const sequence = Array(length).fill(n1)
```

Comment:
Use `new Array` instead of `Array`. It is a potential bug. Why? If you see MDN, you will not find anything about `Array` function.